lib/ansible/modules/windows/win_service.py

```diff
@@ -60,18 +58,24 @@
     description:
       - Whether to allow the service user to interact with the desktop.
       - This should only be set to true when using the LocalSystem username.
-    required: False
     default: False
     version_added: "2.3"
   description:
     description:
       - The description to set for the service.
-    required: False
     version_added: "2.3"
   display_name:
     description:
       - The display name to set for the service.
-    required: False
+    version_added: "2.3"
+  ignore_dependencies:
+    description:
+    - When stopping or restarting a service this will Force the service to stop
```

Comment:
How about "If `true`, stopping or restarting a service with dependent services will force the dependent services to stop or restart also. If `false`, stopping or restarting a service with dependent services may fail.